db/migrate/20220711085009_add_auto_closed_at_to_description_change_validation_requests.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class AddAutoClosedAtToDescriptionChangeValidationRequests < ActiveRecord::Migration[6.1]
+  class DescriptionChangeValidationRequest < ApplicationRecord
+    scope :auto_closed, -> { where(auto_closed: true) }
+  end
+
+  def change
+    add_column :description_change_validation_requests, :auto_closed_at, :datetime
+
+    DescriptionChangeValidationRequest.auto_closed.find_each do |request|
+      request.update(auto_closed_at: request.updated_at)
+    end
```

Comment:
When the closed status was added it seems as if the person missed out the `closed_at` timestamp so we actually have no way of knowing the actual value of this.

Not sure whether we just leave as nil for historical records or assume the updated_at column is correct as usually a closed request does not have further action taken on it